src/XenCert/StorageHandler.py

```diff
@@ -2460,6 +2460,7 @@ class StorageHandlerHBA(BlockStorageHandler):
     def __init__(self, storage_conf):
         XenCertPrint("Reached StorageHandlerHBA constructor")
         super(StorageHandlerHBA, self).__init__(storage_conf)
+        self.sr_type = "lvmo" + self.storage_conf['storage_type']
```

Comment:
@DeliZhangX Updated tests to just use the storageHandlerHBA class and then just create fcoe/hba sr based on the storage_type param. Didn't give me any problems with the controlPathStressTests.

Reply:
OK